markdown.dtx

```diff
@@ -22979,10 +23004,14 @@ parsers.in_matched_block_tags = parsers.less
 %
 % \end{markdown}
 %  \begin{macrocode}
-parsers.hexentity = parsers.ampersand * parsers.hash * S("Xx")
-                  * C(parsers.hexdigit^1) * parsers.semicolon
+parsers.pattern_range = function(pattern, min, max)
+  return -pattern^(max + 1) * pattern^min
+end
```

Comment:
Nitpicks: The function is confusingly named. Furthermore, it's not a parser and therefore should not be part of `parsers`.

```suggestion
local function repeat_between(pattern, min, max)
  return -pattern^(max + 1) * pattern^min
end
```